W/Wayland/build_tarballs.jl

```diff
@@ -3,35 +3,57 @@
 using BinaryBuilder
 
 name = "Wayland"
-version = v"1.21.0"
+version = v"1.23.0"
 
 # Collection of sources required to build Wayland
 sources = [
-    ArchiveSource("https://gitlab.freedesktop.org/wayland/wayland/-/releases/$(version)/downloads/wayland-$(version).tar.xz",
-                  "6dc64d7fc16837a693a51cfdb2e568db538bfdc9f457d4656285bb9594ef11ac"),
+   GitSource("https://gitlab.freedesktop.org/wayland/wayland.git",
+             "a9fec8dd65977c57f4039ced34327204d9b9d779"),
 ]
 
+# Wayland host exectuable (of the same version) is required to cross-compile wayland
+# Need to first build on x86_64-linux (by setting init_bootstrap = true) to then (by setting init_bootstrap = false) build on other platforms
+init_bootstrap = true
```

Comment:
Isn't it easier to do the bootstrap locally for all platforms thank doing this dance?  That'd be cached by ccache, making this process less annoying.